CLI/controllers/layer.go

```diff
@@ -167,11 +168,16 @@ func (controller Controller) UpdateLayer(path string, attributeName string, valu
 	case models.LayerFiltersRemove:
 		_, err = controller.UpdateObj(path, map[string]any{attributeName: value})
 	case models.LayerFiltersAdd:
-		values := strings.Split(value.(string), "=")
-		if len(values) != 2 || len(values[0]) == 0 || len(values[0]) == 0 {
-			return fmt.Errorf("invalid filter format")
+		var filters map[string]any
+		reg := regexp.MustCompile(`^(\w(\w|\-)*)=[^!<>=&|]+$`)
+		if strings.HasPrefix(value.(string), "filter=") {
```

Comment:
Isn't it filters+= the prefix to add filters?

Reply:
It is! Line 170 (`case models.LayerFiltersAdd:`) checks for this. Line 173 checks if the user has specified `filter=` in the complex filter. 
e.g. `test:filters+=filter=category!=room`